File: linear_search.py
def linearSearch(list, target):
  """
  Reaturn The Index Posetion Of Target If It's Found Else Return None
  """
  for i in range(0, len(list)): 
    if list[i] == target:
      return i 
  return None

numbers = [1,2,3,4,5,6,7,8,9]

def verify(index):
  if index is not None :
    print("the target found at index", index)
  else: 
    print("the target not found in the list")
    
result = linearSearch(numbers, 12)
verify(result)
        
result = linearSearch(numbers, 9)
verify(result)
        
        

File: linear-search.js
function linear_search(list, target) {
  for (let i = 0; i <= list.length; i++) {
    if (list[i] == target) {
      return i;
    }
  }
  return null;
}

const items = [1, 2, 3, 4, 5, 6, 7, 8, 9];

function verify(index) {
  if (index != null) {
    console.log(`The target is found in index ${index}`);
  } else {
    console.log("the target is not found ");
  }
}

let result = linear_search(items, 1);

verify(result);
